### PR TITLE
Move Aryn DocParse Docs Out of Sycamore

### PR DESCRIPTION
**Pull request by karanataryn**
Move Aryn DocParse out of Sycamore docs since they have a new home at docs.aryn.ai and https://github.com/aryn-ai/docs

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,66 +1,16 @@
...
+Aryn is an LLM-powered data preparation, processing, and analytics system for complex, unstructured documents like PDFs, HTML, presentations, and more. With Aryn, you can prepare data for GenAI and RAG applications, power high-quality document processing workflows, and run analytics on large document collections with natural language. It includes three components: Aryn DocParse, Aryn DocPrep, and Sycamore. For Aryn DocParse and Aryn DocPrep, please visit the `Aryn DocParse documentation <https://docs.aryn.ai>`_.
```
Comment (jonfritz) [inline docs/source/index.rst:3]: I would remove this paragraph. I would then add: "For Aryn DocParse and Aryn DocPrep, which use Sycamore, please visit the `Aryn documentation <https://docs.aryn.ai>`_." I would put this after the paragraph with Qdrant and more.

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,66 +1,16 @@
...
 
 .. image:: images/ArynArchitecture_APS+Sycamorev2.png
 
-**Key Features**
-
-* **Powerful and accurate document segmentation for complex documents with tables, images, text, infographics, and more.**  Aryn’s vision-based segmentation models provide 11+ class labels, and are up to 6x better in mean average precision (mAP) and 4x better in mean average recall (mAR) than existing solutions.
-
-* **Developer-focused reliable and flexible document processing engine with Sycamore.** Similar to Apache Spark, but for handling and processing unstructured document collections at scale. Easily process document collections using sophisticated data transforms and LLMs, while maintaining overall document lineage using Sycamore’s DocSet abstraction. Create better chunks and extract higher quality metadata, leading to 30% better recall and 2x better accuracy on real-world use cases.
```
Comment (jonfritz) [inline docs/source/index.rst:17]: Keep this bullet point "Developer-focused reliable and flexible document processing engine" but remove "with Sycamore"

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,66 +1,16 @@
...
-
-* **Scalable, fault-tolerant, and reliable loading of vector DBs and search indexes.** Generate vector embeddings using your choice of model, easily build knowledge graphs, and other output formats from your unstructured data. Targets include leading engines like Elasticsearch, OpenSearch, Weaviate, Pinecone, DuckDB, Qdrant and more. Aryn can seamlessly handle millions of documents.
-
-* **Plug-and-Play LLMs.** Use different LLMs for entity extraction, vector embedding, and post-processing steps.
```
Comment (jonfritz) [inline docs/source/index.rst:21]: Keep both of these bullet points

File: docs/source/sycamore/SDK_APIs.rst [hunk 4]
```diff
@@ -1,10 +1,10 @@
-Aryn Partitioning Service APIs
+Aryn SDK APIs
```
Comment (jonfritz) [inline docs/source/sycamore/SDK_APIs.rst:1]: We shouldn't have any Aryn SDK stuff in the Sycamore docs now - should we remove this?

**Review by jonfritz — COMMENTED** [review]
A few suggestions. Thx!

File: docs/source/sycamore/SDK_APIs.rst (hunk 4 above)
Comment (karanataryn) [reply docs/source/sycamore/SDK_APIs.rst]: Removed.

**Review by jonfritz — APPROVED** [review]
LGTM!

**Pull request merged**